src/components/NavigationAccount.vue

```diff
@@ -176,23 +179,30 @@ export default {
 		},
 	},
 	methods: {
-		createMailbox(e) {
-			this.editing = true
+		async createMailbox(e) {
+			this.nameInput = false
+			this.showSaving = true
 			const name = this.createMailboxName
 			logger.info('creating mailbox ' + name)
 			this.menuOpen = false
-			this.$store
-				.dispatch('createMailbox', { account: this.account, name })
-				.then(() => logger.info(`mailbox ${name} created`))
-				.catch((error) => {
-					logger.error('could not create mailbox', { error })
-					throw error
+			try {
+				await this.$store.dispatch('createMailbox', {
+					account: this.account, name,
 				})
-			this.editing = false
-			this.showSaving = false
+			} catch (error) {
+				logger.error('could not create mailbox', { error })
+				throw error
+			} finally {
+				this.showSaving = false
+				this.nameInput = false
+				this.editing = false
+				this.createMailboxName = ''
+			}
+			logger.info(`mailbox ${name} created`)
```

Comment:
I don't think these are supposed to be backticks instead of apostrophes

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals